lib/menu.rb

```diff
@@ -0,0 +1,33 @@
+require 'yaml'
+
+class Menu
+
+  def initialize(menu_file = 'menu.yml')
+    @menu = YAML::load(File.open('./lib/resources/'+menu_file))
+  end
+
+  def view
+    "// MENU //"+print_menu
+  end
+
+  def find(dish_name)
+    @menu.each do |section, dishes|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `section`. If it's necessary, use `_` or `_section` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)